frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/tags/DisplayReleaseClearingStateSummary.java

```diff
@@ -49,13 +31,14 @@ public void setReleaseClearingStateSummary(ReleaseClearingStateSummary releaseCl
     }
 
     public void doTag() throws JspException, IOException {
-        String s = "<span title=\"new release, under clearing, under clearing by the project clearing team, report available, approved\">" +
-                releaseClearingStateSummary.newRelease + " " +
-                releaseClearingStateSummary.underClearing + " " +
-                releaseClearingStateSummary.underClearingByProjectTeam + " " +
-                releaseClearingStateSummary.reportAvailable + " " +
-                releaseClearingStateSummary.approved +
-                "</span>";
+        String releaseCounts;
+        if (releaseClearingStateSummary == null) {
+            releaseCounts = "not available";
+        } else {
+            releaseCounts = releaseClearingStateSummary.newRelease + " " + releaseClearingStateSummary.underClearing + " " + releaseClearingStateSummary.underClearingByProjectTeam + " " + releaseClearingStateSummary.reportAvailable + " " + releaseClearingStateSummary.approved;
+        }
+
+        String s = "<span title=\"new release, under clearing, under clearing by the project clearing team, report available, approved\">" + releaseCounts + "</span>";
         getJspContext().getOut().print(s);
```

Comment:
If there is no releaseClearingStateSummary, shouldn't then also the title be empty?